turbo.json

```diff
@@ -20,10 +16,18 @@
     "test:components": {
       "outputs": ["playwright-report", "test-results"]
     },
+    "sit-onyx#test:components": {
+      "dependsOn": ["sit-onyx#build"]
+    },
```

Comment:
Why are the tests dependent on the build? Playwright builds the components separately

Reply:
You are right, what I wanted to achieve was "tests depend on the build of dependencies". Should be fixed thorugh the last commit.